cmd/lotus-storage-miner/market.go

```diff
@@ -569,6 +572,67 @@ var dataTransfersCmd = &cli.Command{
 	Usage: "Manage data transfers",
 	Subcommands: []*cli.Command{
 		transfersListCmd,
+		minerRestartTransfer,
+	},
+}
+
+var minerRestartTransfer = &cli.Command{
+	Name:  "restart-transfer",
```

Comment:
this can just be "restart" rather than "restart-transfer" cause the miner command will already be under the "data-transfers" namespace